src/backend/expungeservice/record_creator.py

```diff
@@ -149,11 +149,33 @@ def _edit_search_results(search_result_cases: List[OeciCase], edits) -> Tuple[Li
             if case_number in edits.keys():
                 if edits[case_number]["action"] == "edit":
                     edited_case, new_charges = RecordCreator._edit_case(case, edits[case_number])
-                    edited_cases.append(edited_case)
+                    edited_cases.append(replace(edited_case, summary=replace(edited_case.summary, edit_status=EditStatus.EDITED)))
```

Comment:
This should be done in edit case:

`edited_summary = replace(case.summary, edit_status=EditStatus.EDITED, **case_summary_edits)`